lib/ferver/app.rb

```diff
@@ -10,6 +10,14 @@ class App < Sinatra::Application
       set :views, './views'
     end
 
+    use Rack::Auth::Basic, 'Ferver' do |username, password|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.